pkg/consts/consts.go

```diff
@@ -64,12 +64,14 @@ const (
 	// MaxEvents is the maximum number of events we can parse in a single batch.
 	MaxEvents = 5_000
 
+	InngestEventDataPrefix = "_inngest"
 	// InvokeEventName is the event name used to invoke specific functions via an
 	// API.  Note that invoking functions still sends an event in the usual manner.
 	InvokeEventName = "inngest/function.invoked"
 	// InvokeSlugKey is the data key used to store the fn name when invoking a function
 	// via an RPC-like call, abstracting event-driven fanout.
-	InvokeSlugKey = "_inngest_fn"
+	InvokeFnID          = "fn_id"
+	InvokeCorrelationId = "correlation_id"
```

Comment:
Good ol' MQ.  It's good to stick to the standards — great idea.

Reply:
Mostly as we can have many invocations from one run, so anything we have in the triggering event (event ID, run ID) still needs more data adding to it to be meaningful.